### ISSUE TITLE
Icon to view edit history is not a true button

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#715]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.2 (A) Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value).

In the chat, if a message has been edited, there's a clickable icon that brings up the message edit history. While this functions as a button in relation to pointing devices, it's not accessible by keyboard or assistive technologies.

![The icon that indicates a message has been edited, and its attributes as seen in Chrome devtools.](https://github.com/user-attachments/assets/13c521be-d26d-4a52-95ef-2017de36a5aa)

**To Reproduce**
1. Send a message in chat.
2. Edit that message.
3. Inspect the icon that indicates an edited message, or attempt to interact with it using the keyboard or assistive technology (for example a screen reader).

**Expected behavior**
Since this icon functions as a button, it can be expected that it should work as a button. It should be reachable by mouse, keyboard and assistive technologies, and have an accessible name.

**Suggestion**
Try to use native HTML elements where possible. In this case simply using a <button> would be ideal.

Remember that buttons without a visible text label need an accessible name through other means. Many of the buttons in the chat widget use the `title` attribute for this, which is mainly meant to show up on mouse hover and isn't really the ideal approach for this (although many screen readers will handle it well enough). We recommend using `aria-label` for this purpose instead, possibly in conjunction with `title` if you want it to visibly appear on hover.

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 12.0.3

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
